### ISSUE TITLE
Script error

### ISSUE THREAD

**Issue by four2six** [allan-gam/GCal#17]
Events aren't fetched anymore.

```
Debug is on
Google Calendar API for Domoticz Version 0.0.4 running on TTY console...
Found a total of : 1 calendar configuration entries.

Fetching Calendar : Google Calendar. Last fetched: 5 minutes ago. Interval: 1
Getting events for calendar xxx@group.calendar.google.com
(u'2017-04-28T19:30:00+02:00', u'nacht')
(u'2017-04-28T22:30:00+02:00', u'nacht1')
(u'2017-04-28T22:30:00+02:00', u'1nacht1')
Traceback (most recent call last):
  File "/home/pi/GCal/gcal.py", line 820, in <module>
    main(sys.argv[1:])
  File "/home/pi/GCal/gcal.py", line 816, in main
    list_calendars()
  File "/home/pi/GCal/gcal.py", line 760, in list_calendars
    process_calendar(c)
  File "/home/pi/GCal/gcal.py", line 665, in process_calendar
    if not keyWordFound: keyWordFound = e['description'].lower().find(keyWord) > 0
KeyError: 'description'
```

**Comment by four2six** [comment]
i should have used the sd card method :))))

**Comment by allan-gam** [comment]
I've made an update. Does it resolve the problem?

**Comment by four2six** [comment]
no
the text device is not updated

even though: 
```
Debug is on
Google Calendar API for Domoticz Version 0.0.5 running on TTY console...
Found a total of : 1 calendar configuration entries.

Fetching Calendar : Google Calendar. Last fetched: 29 minutes ago. Interval: 1
Getting events for calendar c1fgnon19pl67h72mrmt9dtb5c@group.calendar.google.com
(u'2017-04-28T19:30:00+02:00', u'nacht')
(u'2017-04-28T22:33:00+02:00', u'nacht1')
(u'2017-04-28T22:35:00+02:00', u'1nacht1')
GCal Status Google Calendar
N: <span style="color: grey;">No future events found  !#0(0)!#</span>
D: <span style="color: grey;">No future events found  !#0(0)!#</span>

GCal Google Calendar
N: Off
D: Off

Config changed... Needs to save
```

**Comment by four2six** [comment]
i set up a new device with the same calendar... but without the keyword. 
the keyword one doesn't update the device, the one without does.

**Comment by four2six** [comment]
oh damn, i made a mistake. i searched for the wrong keyword. too muchg testing, sorry. 

BUT it's not showing the next event, but the last (keyword is 'nacht'). and it misses the from...to.. attributes

![bildschirmfoto 2017-04-28 um 13 31 41](https://cloud.githubusercontent.com/assets/4852434/25527071/21277304-2c17-11e7-978b-f76f8bb67a62.png)
![bildschirmfoto 2017-04-28 um 13 32 02](https://cloud.githubusercontent.com/assets/4852434/25527073/232b83a2-2c17-11e7-9b8e-4e7312ceeb67.png)

**Comment by allan-gam** [comment]
I Found the problem and I have fixed it in version 0.0.8

**Comment by four2six** [comment]
yes, that did it, well done.
sorry for the confusion.
